src/Umbraco.Web.UI.Client/src/views/common/overlays/mediaPicker/mediapicker.controller.js

```diff
@@ -133,8 +133,7 @@ angular.module("umbraco")
                     });
                 $scope.currentFolder = folder;
 
-                // for some reason i cannot set cookies with cookieStore
-                document.cookie = "umbLastOpenedMediaNodeId=" + folder.id;
+                localStorageService.set("umbLastOpenedMediaNodeId", folder.id);

```

Comment:
local storage is persistent for ALL users, so this will certainly lead to problems since this data will be shared between all users that log in right? On a side note the previous code:

```
document.cookie = "umbLastOpenedMediaNodeId=" + folder.id;
```

is crazy (has it been like this the whole time?), that would basically replace all cookies with null and just add one :/